.stylelintrc.js

```diff
@@ -1,5 +1,11 @@
+/** @type {import('stylelint').Config} */
 module.exports = {
   extends: ['@shopify/stylelint-plugin/prettier', './stylelint-polaris'],
+  ignoreFiles: [
```

Comment:
Can we put these ignores into https://github.com/Shopify/polaris/blob/main/.stylelintignore?

Reply:
Discussed offline, stylelint is not leveraging the root level `.stylelintignore` with the recent update to use package level run-scripts, but does pick up the root level config. Thus, we decided to remove the `.stylelintignore` file in favor of the `ignoreFiles` config: https://github.com/Shopify/polaris/pull/6445